### PR TITLE
removed reporter.pl from repo. Never really used.

### PR DESCRIPTION
**Pull request by Takadonet**
Address #6

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changes
 
+## 1.8.X
```
Comment (apetkau) [inline CHANGELOG.md]: Since we haven't released `1.8.3`, you can just merge the message from `1.8.3` into `1.8.X`.

**Review by apetkau — CHANGES_REQUESTED** [review]
Looks great. Thanks @Takadonet. One comment.

File: CHANGELOG.md (hunk 1 above)
Comment (Takadonet) [reply CHANGELOG.md]: Done

**Review by apetkau — APPROVED** [review]

**Comment by apetkau** [comment]
Looks good. Thanks.

**Pull request merged**